components/api_server/src/healthcheck.py

```diff
@@ -5,7 +5,7 @@
 from urllib.request import urlopen
 import sys
 
-api_server_port = getenv("API_SERVER_PORT", "5001")
+api_server_port = getenv("API_SERVER_PORT")
```

Comment:
Might be better to then `os.environ["API_SERVER_PORT"]`, as `getenv` is essentially a `os.environ.get` which returns `None` instead of throwing an exception when the var isn't defined